network-api/networkapi/buyersguide/templates/bg_base.html

```diff
@@ -28,7 +28,8 @@
         </div>
       </div>
     </header>
-    <div class="primary-nav">
+
+    <div class="primary-nav sticky-top bg-white">
```

Comment:
Add webkit sticky to the CSS for this bit manually?

Reply:
going with @mmmavis's recommendation!